assets/package.json

```diff
@@ -18,7 +18,7 @@
     "phoenix": "../deps/phoenix",
     "phoenix_html": "file:../deps/phoenix_html",
     "phoenix_live_view": "file:../deps/phoenix_live_view",
-    "uplot": "^1.0.9"
+    "uplot": "^1.0.11"
```

Comment:
Please revert the changes to assets as they are unrelated to this PR?

Reply:
Not unrelated, at least with the current behavior, as i mentioned this fixes a bug when the events are sent quickly, though with above comment I guess a goal is the change that, so if that's successful it might not be needed when it's done.

Reply:
this fixed a bug with auto-scaling not kicking in until the second setData call (leaving the scales at their data-less 0..1 after the first setData call). there was no "data sent too quickly" bug fixed by this.

Reply:
Cool. I can fix this locally, thanks! But we cannot accept user committed priv/ files for security reasons. So it is best to revert the assert changes in general. :)

Reply:
Pushed. Feel free to rebase.